README.md

```diff
@@ -7,20 +7,52 @@ This project is a loader to run the [Hermit kernel](https://github.com/hermitcor
 * [`rustup`](https://www.rust-lang.org/tools/install)
 * [NASM](https://nasm.us/) (only for x86_64)
 
-## Building
+### UEFI Boot (x86_64 only)
+
+QEMU does not support UEFI as firmware interface per se. You'll need to install [OVMF](https://github.com/tianocore/tianocore.github.io/wiki/OVMF-FAQ#what-is-open-virtual-machine-firmware-ovmf) (an open source implementation of UEFI for virtual machines).
+You can do so, e.g., via terminal:
+```bash
+$ sudo apt install ovmf
+```
```

Comment:
Can you also put a pointer to the pre-built RPMs? :)

https://www.kraxel.org/repos/jenkins/edk2/

Extracting works with [`rpm2cpio`](https://www.man7.org/linux/man-pages/man8/rpm2cpio.8.html):

```bash
rpm2cpio edk2.git-ovmf-x64-*.rpm | cpio -idmv
```

Reply:
This one liner works:

```bash
wget -O - "http://ftp.debian.org/debian/pool/main/e/edk2/ovmf_2023.11-6_all.deb" | tar xvf - -O data.tar.xz | tar xvf - --strip-components=4 ./usr/share/OVMF/OVMF_CODE_4M.fd ./usr/share/OVMF/OVMF_VARS_4M.fd && mv OVMF_CODE_4M.fd OVMF_CODE.fd && mv OVMF_VARS_4M.fd OVMF_VARS.fd
```